docs/markdown/connecting-to-testnet.md

```diff
@@ -0,0 +1,58 @@
+# Connecting to an ILP Testnet
+
+This tutorial will help you start an ILP node locally on your computer and connect to an ILP testnet node.
+
+This tutorial describes how to:
+
+1. Configure [`moneyd`](https://github.com/interledgerjs/moneyd) to connect to a node on testnet
+2. Start an [ILP node](https://github.com/interledgerjs/ilp-connector) locally using moneyd
```

Comment:
I think we could combine #1 and #2 to be just: 

1.  Configure and start [`moneyd`](https://github.com/interledgerjs/moneyd) to connect to a Connector on the testnet.

Reply:
https://github.com/interledger/interledger.github.io/pull/108/commits/7fe7c009db48ec2dc6b03b772041d7173b9d010d#diff-95b449acc2e692458b91372a5b6e33e7R7